component/file_cache/file_cache.go

```diff
@@ -966,6 +966,9 @@ func (fc *FileCache) OpenFile(options internal.OpenFileOptions) (*handlemap.Hand
 	handle := handlemap.NewHandle(options.Name)
 	handle.SetValue("openFileOptions", openFileOptions{flags: options.Flags, fMode: options.Mode})
 	// Increment the handle count in this lock item as there is one handle open for this now
+	if options.Flags&os.O_APPEND != 0 {
+		handle.Flags.Set(handlemap.HandleOpenedAppend)
+	}
```

Comment:
This looks good, but you split a comment (above) from its code (below).

Reply:
Fixed.